docs/en/04_Changelogs/4.0.0.md

```diff
@@ -217,152 +127,7 @@ admin/images/sprites/dist/ => admin/client/dist/sprites/
-
-The following methods and properties on `Requirements_Backend` have been renamed:
-
- * `Requirements_Backund::$combine_files` made protected and renamed `$combinedFiles`
- * `Requirements_Backend::$combine_js_with_min` made protected and renamed `$minifyCombinedFiles`
- * `Requirements_Backend::$write_header_comments` made protected and renamed `$writeHeaderComment`
- * `Requirements_Backend::$write_js_to_body` made protected and renamed to `$writeJavascriptToBody`
- * `Requirements_Backend::$force_js_to_bottom` renamed to `$forceJSToBottom`
- * `get_combined_files_enabled` renamed to `getCombinedFilesEnabled`
- * `set_combined_files_enabled` renamed to `setCombinedFilesEnabled`
- * `get_suffix_requirements` renamed to `getSuffixRequirements`
- * `set_suffix_requirements` renamed to `setSuffixRequirements`
- * `get_custom_scripts` renamed to `getCustomScripts`
- * `unblock_all` renamed to `unblockAll`
- * `include_in_response` renamed to `includeInResponse`
- * `combine_files` renamed to `combineFiles`
- * `get_combine_files` renamed to `getCombinedFiles`
- * `clear_combined_files` renamed to `clearCombinedFiles`
- * `process_combined_files` renamed to `processCombinedFiles`
- * `set_write_js_to_body` renamed to `setWriteJavascriptToBody`
- * `set_force_js_to_bottom` renamed to `setForceJSToBottom`
-
-New methods on `Requirements` are added to access these:
-
- * `get_minify_combined_js_files`
- * `set_minify_combined_js_files`
- * `get_force_js_to_bottom`
- * `get_write_js_to_body`
- 
-Some methods on `Requirements` have had their method signatures changed:
-
- * `includeInHTML` has had the first parameter $template removed as it was previously deprecated.
-
-And some methods on `Requirements` and `Requirements_Backend` have been removed as they are obsolete.
-
- * `delete_combined_files` (both classes)
-
-A new config `Requirements_Backend.combine_in_dev` has been added in order to allow combined files to be
-forced on during development. If this is off, combined files is only enabled in live environments.
-
-In addition, a new API for javascript files has been added to support front-end tools
-such as [browserify](http://browserify.org/) that pre-combine scripts. You can specify
-a 'provides' option to the second parameter of `Requirements::javascript` to declare
-included files.
-
-E.g.
-
-
-    :::php
-    Requirements::javascript('mysite/js/dist/bundle.js', ['provides' => [
-        'mysite/js/jquery.js'
-        'mysite/js/src/main.js',
-        'mysite/js/src/functions.js'
-    ]]);
-    
-### RestfulService
-
-* `RestfulService` has been removed. Use Guzzle instead. See Upgrading notes.
-
-### Oembed
-
-* Our self-maintained `Oembed` implementation has been removed, in favour of introducing [oscarotero/Embed](https://github.com/oscarotero/Embed) as a dependency. 
-
-
-## Upgrading
-
-### Explicit text casting is now enforced on all template variables
+#### Explicit text casting is now enforced on all template variables
```

Comment:
I'd simplify the before/after example to just one example, with a PHP comment pointing out the new bit (`$casting`)